nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/parameter-contexts/ui/parameter-context-listing/parameter-context-table/parameter-context-table.component.html

```diff
@@ -54,6 +54,16 @@
                 </td>
             </ng-container>
 
+            <!-- Bounded Process Group Column -->
+            <ng-container matColumnDef="bounded process group">
```

Comment:
I recommend just `process groups`, or perhaps `bound process groups`.
```suggestion
            <ng-container matColumnDef="process group">
```